EIPS/eip-3475.md

```diff
@@ -1,428 +1,605 @@
 ---
 eip: 3475
 title: Multiple Callable Bonds Standard
-description: A standard interface for contract that manage multiple bonds. This standard will allow multiple LP token contract to store more information
+description: An interface for contracts that manage multiple bonds, allowing multiple LP token contracts to store more information.
 author: Yu LIU (@yuliu-debond), Waroon D. (@dr-chain), Cedric Ngakam (@drikssy), Dhruv Malik (@GrandGarcon), Samuel Gwlanold (@Edoumou), Toufic Batrice (@toufic0710)
 discussions-to: https://ethereum-magicians.org/t/eip-3475-multiple-callable-bonds-standard/8691
-status: Review
+status: Last Call
+last-call-deadline: 2022-07-30.
```

Comment:
```suggestion
last-call-deadline: 2022-07-30
```

Reply:
done. sorry last time i had resolved that but some changes but they came again.